x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -46,12 +46,15 @@ export const PROCESSOR_EVENT = 'processor.event';
 
 export const TRANSACTION_DURATION = 'transaction.duration.us';
 export const TRANSACTION_DURATION_HISTOGRAM = 'transaction.duration.histogram';
+export const TRANSACTION_DURATION_SUMMARY = 'transaction.duration.summary';
```

Comment:
iirc we talked about splitting this file into several files, with one file per data source (eg. APM vs infra vs logs) we might also want to split the fields by doc type (transactions vs transaction metrics vs service metrics)